common/mariadb-galera/helm/tests/02-services-mariadb-galera_test.yaml

```diff
@@ -974,3 +974,37 @@ tests:
       - equal:
           path: spec.sessionAffinityConfig.clientIP.timeoutSeconds
           value: 28800
+  - it: database service name changed if migration option is enabled
+    template: service.yaml
+    documentSelector:
+      path: metadata.name
+      value: testrelease-mariadb-replica
+    set:
+      mariadb.galera.clustername: "testclustername"
+      mariadb.migration.enabled: true
+    asserts:
+      - isKind:
+          of: Service
+      - equal:
+          path: metadata.name
+          value: testrelease-mariadb-replica
+      - equal:
+          path: metadata.labels.app
+          value: testrelease
+  - it: database service name changed if migration option is disabled
```

Comment:
please add one empty line in between the tests ([it]s)

Reply:
if i got it correctly you are testing that the replica service is not created .. for that i think it is better to check that the name of the service, in the document index (where the replica service should be created if not working) , is not equal to (testrelease-mariadb-replica )

Reply:
I have added `notEqual` checks too